google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb

```diff
@@ -173,10 +173,12 @@ def retrieve_snapshot project, subscription, snapshot_name
       next_subs.token.must_be :nil?
     end
 
-    it "should allow creation of a subscription" do
-      subscription = topic.subscribe "#{$topic_prefix}-sub3"
+    it "should allow creation of a subscription with options" do
```

Comment:
@blowmage Actually, I just noticed that [other tests](https://github.com/quartzmo/google-cloud-ruby/blob/49d37fe5321c7d79a5b02ab45ee9ea8e7ad9b915/google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb#L191-L214) create subscriptions without options, so it's covered.